CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## Unreleased
 
+### Recommended changes
+
+#### Remove `aria-labelledby`, remove `id="error-summary-title"` from title and move `role="alert"` to child container on the error summary component
+
+If you're not using the Nunjucks macros, we recommend making the following changes to the error summary markup:
```

Comment:
Yeah, this looks alright to me. I'd suggest giving the reason for the recommendation earlier on.

Also, is "the Nunjucks macros" clear enough? I can't tell from this if it's 'all Nunjucks macros in GOV.UK Frontend' or just the ones in the error summary component.

```suggestion
If you're not using the Nunjucks macros, you can help screen reader users by making these changes to the error summary markup:
```

Reply:
Will make the change in a mo. I took the nunjucks macros line as a convention in our changelog. Generally if a team aren't using one of our macros then it's very likely they aren't using our macros at all as typically the reason to not use our macros is a question of codebase compatibility eg: govuk using ruby on rails which doesn't fit with nunjucks. It's a bit tricky to convey though so I'm open to suggestions.